Challenge3.py

```diff
@@ -0,0 +1,54 @@
+class Book():
+    def __init__(self, title, Author, Isbn, Genre, Copies):                 #BookClass
+        self.Title = title
+        self.Author = Author
+        self.Isbn = Isbn
+        self.Genre = Genre
+        self.Copies = Copies
+
+        #Author Class
+class Author():
+    def __init__(self, Auth_name, Auth_birthday , Auth_nationality):
+        self.Auth_name = Auth_name
+        self.Auth_birthday = Auth_birthday
+        self.Auth_nationality = Auth_nationality
+        
+        #library Class Catalog
+class LibraryCatalog():
+    def __init__(self):
+        self.Book = []       #empty list to store books
```

Comment:
Use attributes and function in lowercase
```suggestion
        self.book_list = []       #empty list to store books
```